plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/glue/DeltaLakeGlueMetastoreTableFilterProvider.java

```diff
@@ -51,7 +51,6 @@ private static boolean isDeltaLakeTable(Table table)
         if (parameters == null) {
             return false;
         }
-        // todo; add parameters == null check to DefaultGlueMetastoreTableFilterProvider.isDeltaLakeTable (https://github.com/trinodb/trino/issues/12013)
```

Comment:
remove `Map<String, String> parameters = table.getParameters();` above too

Reply:
Why ? To me it is a 'fast path' for returning false

Reply:
it's a code piece that is now redundant. we wouldn't write it originally if `DefaultGlueMetastoreTableFilterProvider.isDeltaLakeTable` didn't have a problem

actually, the whole method should get inlined now